src/webgpu/wgsl_language_feature.ts

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
Usually we just add new things like this to `capability_info.ts`. This will stay small enough that I'd say just put it there.